File: src/main/webapp/osca/common/domain/Referentiel/Rubriques/SousRubriques/Fonc/FeuilleDeRouteFonc.ts
import { CriteresFonc } from './CriteresFonc';
import { PratiquesFonc } from './PratiquesFonc';

export interface FeuilleDeRouteFonc {
    titre: string;
    keyFeuilleDeRouteFonc: string;
    criteresFoncList: CriteresFonc[];
    pratiquesFoncList: PratiquesFonc[];
}

File: src/main/java/com/soprasteria/osca/domain/referentiel/rubriques/sousrubriques/fonc/FeuilleDeRouteFonc.java
package com.soprasteria.osca.domain.referentiel.rubriques.sousrubriques.fonc;

import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class FeuilleDeRouteFonc {

    private String titre;
    private String keyFeuilleDeRouteFonc;
    private List<CritereFonc> criteresFoncList;
    private List<PratiqueFonc> pratiquesFoncList;
}
